{"name": "Проверка авторизации пользователем с неверным паролем", "status": "passed", "start": 1719861605412, "stop": 1719861605777, "uuid": "dcdcdccc-4ebc-4650-a9d0-0b88d9d34a74", "historyId": "b1989d319aef0a9eec9347f59afe5406", "testCaseId": "b1989d319aef0a9eec9347f59afe5406", "fullName": "tests.test_user_authentication.TestLoginUser#test_login_user_incorrect_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_authentication"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-6FRR489"}, {"name": "thread", "value": "16736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_authentication"}]}